app/src/main/java/com/android/vending/billing/utils/Base64.java

```diff
@@ -31,6 +31,8 @@
  * @version 1.3
  */
 
+import com.simplecity.amp_library.BuildConfig;
```

Comment:
This is a third party library, prefer not to make changes here, as it makes it difficult to merge in future changes to this library.